### DISCUSSION TITLE
Add source for DE_Schmalkalden-Meiningen

### DISCUSSION THREAD

**Discussion by chriscomputerfreak** [mampfes/hacs_waste_collection_schedule#590] [Ideas]
Hello, 
can you please add source for Schmalkalden Meiningen in Germany?
http://www.schmalkalden-meiningen.hausmuell.info/
Thank you very much :-)

**Comment by 5ila5** [comment]
fixed by #851

**Comment by Maqsyo** [comment]
Hey @5ila5 leider gibt es hier ein Problem mit der Stadt "Zella-Mehlis", da diese 3 Ortsteile hat.
Unter der API https://schmalkalden-meiningen.hausmuell.info/search/search_orte.php kommt Zella-Mehlis mit der ort-id `10457` zurück.
Der Ortsteil `Stadt` hat die id `10458`

Sucht man nun nach Straßen, so wird im Python-Skript die ort-id und nicht die ortsteil-id genommen.
unter https://schmalkalden-meiningen.hausmuell.info/search/search_strassen.php liefert die ort-id von Zella-Mehlis `10457` eine leere Liste zurück.
Die Ort-Id von dem Ortsteil hingegen (`10458`) liefert die Straßennamen zurück.
Dadurch kommt es zum Fehler `"strasse provided but not found in search results."`.

Im Python-Skript gibt es auch nur ein Beispiel mit einem Orsteil, aber ohne Straßen
https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/custom_components/waste_collection_schedule/waste_collection_schedule/source/hausmuell_info.py#L48
Lasse ich Straße und Hausnummer in meinem Fall weg, kommt es jedoch dennoch zum Fehler

eine Anpassung müsste also hier erfolgen:
https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/custom_components/waste_collection_schedule/waste_collection_schedule/source/hausmuell_info.py#L242

Alternativ wäre eine direkte Eingabe von den IDs sinnvoll :)

**Reply by 5ila5** [reply]
transferred to issue #1122 
fixed by #1130

Fix will be part of release 1.42.0 or is available now as master/git branch.
